enterprise/users/src/test/java/io/crate/integrationtests/PrivilegesIntegrationTest.java

```diff
@@ -350,4 +354,19 @@ public void testOperationOnClosedTableAsAuthorizedUser() {
                                                        "is currently closed."));
         execute("refresh table s.t1", null, testUserSession());
     }
+
+    @Test
+    public void testPermissionsValidOnTableAlias() {
+        executeAsSuperuser("create table s.t1 (x int)");
+        executeAsSuperuser("insert into s.t1 values (1)");
+        executeAsSuperuser("insert into s.t1 values (2)");
```

Comment:
IMO, no need to insert three records. the test should also work without any rows in the table, right?

Reply:
Good point! Yeah, it should.